ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-upi-e2e.yaml

```diff
@@ -0,0 +1,1313 @@
+kind: Template
+apiVersion: template.openshift.io/v1
+
+parameters:
+- name: JOB_NAME
+  required: true
+- name: JOB_NAME_SAFE
+  required: true
+- name: JOB_NAME_HASH
+  required: true
+- name: NAMESPACE
+  required: true
+- name: IMAGE_TESTS
+  required: true
+- name: IMAGE_OPENSTACK_INSTALLER
+  required: true
+- name: CLUSTER_TYPE
+  value: "openstack"
+- name: TEST_COMMAND
+  required: true
+- name: RELEASE_IMAGE_LATEST
+  required: true
+- name: OS_CLOUD
+  value: openstack-cloud
+- name: OPENSTACK_EXTERNAL_NETWORK
+  value: external
+- name: RHCOS_RELEASE
+  value: master
+- name: OPENSTACK_FLAVOR
+  value: m1.s2.xlarge
```

Comment:
Do we need this large of a vm for testing at this point?  Until we turn on parallel testing or serial testing. We just need the cluster to come up.